netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -320,7 +321,6 @@ private void sendResponseHeaders(ChannelHandlerContext ctx, SendResponseHeadersC
 
   private void cancelStream(ChannelHandlerContext ctx, CancelServerStreamCommand cmd,
       ChannelPromise promise) {
-    cmd.stream().abortStream(cmd.reason(), false);
```

Comment:
Why remove abortStream? It seems like it is needed to call the listener to notify the application.

I can't find where we listen for streamInactive. Do we no longer do that?